### PR TITLE
[FIX] account : fixed a typo

### PR DESCRIPTION
**Pull request by casp-odoo**

### CODE REVIEW

**Comment by Yenthe666** [comment]
update `.pot` file, no?

**Comment by tsb-odoo** [comment]
indeed

File: addons/account/i18n/account.pot [hunk 1]
```diff
@@ -7367,6 +7367,11 @@ msgstr ""
 msgid "Journal group are used in reporting to display relevant data together."
 msgstr ""
 
+#. module: account
+#: model_terms:ir.actions.act_window,help:account.action_account_journal_group_list
+msgid "Journal groups are used in reporting to display relevant data together."
+msgstr ""
+
```
Comment (MiquelRForgeFlow) [inline addons/account/i18n/account.pot:7374]:
```suggestion
msgid "Journal groups are used in reporting to display relevant data together."
msgstr ""
```
why duplication?

Comment (oco-odoo) [reply addons/account/i18n/account.pot:7374]: @MiquelRForgeFlow That's our standard way of handling translation fixes in stable versions. Updating the existing term directly could break dbs relying on it, so we keep it and add a new, correct one.

Comment (MiquelRForgeFlow) [reply addons/account/i18n/account.pot:7374]: oh, right. thanks!

**Review by smetl — APPROVED** [review]
@robodoo r+

**Comment by smetl** [comment]
@robodoo r-
Unfortunately, this typo is already there in 14.0:
https://github.com/odoo/odoo/blob/14.0/addons/account/views/account_journal_views.xml#L245
Then, this should be fixed in another PR targeting 14.0

**Comment by Yenthe666** [comment]
@smetl I believe @mart-e is usually not a fan of doing this in lower versions and since V15 is quite new this should still be ok I believe :)

**Comment by mart-e** [comment]
Indeed, breaking translations is still tolerated when close to the release. I was not aware of the woraround by adding the new string without removing the previous one but I guess it should work :)

**Pull request closed without merging**